cms/static/js/spec/views/previous_video_upload_list_spec.js

```diff
@@ -99,6 +107,120 @@ define(
                 // Verify that nothing changes when we click on Cancel button on confirmation popup
                 verifyVideosInfo(this, false);
             });
+
+            describe('PagingPreviousVideoUploadListView', function() {
+
+                var videoView, collection;
+                beforeEach(function() {
+                    collection = new VideoPagingCollection([
+                            {
+                                'client_video_id': 'foo.mp4',
+                                'duration': 42,
+                                'created': '2014-11-25T23:13:05',
+                                'edx_video_id': 'dummy_id_1',
+                                'status': 'uploading'
+                            },
+                            {
+                                'client_video_id': 'foo.mp4',
+                                'duration': 42,
+                                'created': '2014-11-25T23:13:05',
+                                'edx_video_id': 'dummy_id_2',
+                                'status': 'uploading'
+                            }
+                        ],
+                        {
+                            url: videoHandlerUrl,
+                            pageSize: 2,
+                            sortField: 'Name',
+                            totalCount: 3,
+                            sortDir: 'asc'
+                        }
+                    );
+                    videoView = new PreviousVideoUploadListView({
+                        collection: collection,
+                        videoHandlerUrl: videoHandlerUrl
+                    });
+
+                    videoView.render();
+                });
+
+                var firstPageResults = {
```

Comment:
`firstPageResults` and `secondPageResults` variables needs to declared where `videoView, collection` are, on the very first line of the function.